lib/fencing/st_rhcs.c

```diff
@@ -289,7 +297,19 @@ stonith__rhcs_validate(stonith_t *st, int call_options, const char *target,
 
     rc = stonith__execute(action);
     if (rc == pcmk_ok) {
-        stonith__action_result(action, &rc, output, error_output);
+        pcmk__action_result_t *result = stonith__action_result(action);
```

Comment:
ok seems to be just one use-case (and maybe that goes away with subsequent patches - didn't check) but otherwise
one could have thought of functionalizing and keeping around something like stonith__action_legacy_result.

Reply:
Eventually, everything will be adapted to use the new codes, and I tried to make it so the diff for that will be easier to review (i.e. just drop the mapping call)